test-trusted-directory/kustomize/overlay/dev-cdk/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
-namespace: nightly-core
+namespace: dev-cdk-core
```

Comment:
and again...
git compare confused again?

Reply:
And Yup!